pkgs/helm-with-plugins.nix

```diff
@@ -0,0 +1,33 @@
+{ pkgs
+, stdenv
+, lib
+, makeWrapper
+, plugins
+, kubernetes-helm
+}:
+
+let
+
+  installPlugin = name: plugin: ''
+    mkdir -p $out/${plugin.pname}
+    ln -s ${plugin}/bin $out/${plugin.pname}
```

Comment:
why don't you just mv :-) ?

Reply:
Because this implicitly puts the plugin as a dependency of the current drv as the plugin might have dependencies itself

Reply:
Moreover you can't `mv` a file of an existing derivation as the nix sotre is RO. You could `cp` but you loose all dependency information